test/helper.lua

```diff
@@ -40,4 +46,53 @@ end
 
 helper.cartridge_cmd = build_cli_binary()
 
+function helper.check_pid_running(pid)
+    return ffi.C.kill(tonumber(pid), 0) == 0
+end
+
+function helper.wait_process_exit(pid, timeout)
+    timeout = timeout or 2
+    if type(pid) == 'table' then
+        pid = tonumber(pid.pid)
+    end
+    local started_at = clock.time()
+    while helper.check_pid_running(pid) do
+        if clock.time() - started_at > timeout then
+            error('expected process to exit, but it does not')
+        end
+        fiber.sleep(0.1)
+    end
+end
+
+-- Non-blocking os.execute() which fails if process does not exit.
+function helper.os_execute(path, args, opts)
+    opts = opts or {}
+    local env = fun.chain(os.environ(), opts.env or {}):tomap()
+
+    local process = Process:start(fio.abspath(path), args, env, {
+        chdir = opts.chdir,
+    })
+    helper.wait_process_exit(process, opts.timeout)
+    return process
+end
+
+function helper.read_file(path)
+    local file = assert(fio.open(path))
+    local result = assert(file:read())
+    file:close()
+    return result
+end
+
+function helper.write_file(path, content)
+    local mode = tonumber(755, 8)
+
+    local file = assert(fio.open(path, {'O_CREAT', 'O_WRONLY', 'O_TRUNC', 'O_SYNC'}, mode))
+    assert(file:write(content))
```

Comment:
I think it should be handled more properly. Without assert

Reply:
This is just a test helper.

Reply:
Ok